tests/conftest.py

```diff
@@ -89,3 +89,16 @@ def _snapshot(**kwargs):
             yield snapshot
 
     return _snapshot
+
+
+@pytest.fixture
+def appsec():
+    """
+    Fixture for the AppSec module that disables it after each test.
```

Comment:
Is there a way to have a singleton for each test rather than a shared object?
This would possible for example to run tests in parallel one day.

Reply:
Ack, will do

Reply:
https://github.com/DataDog/dd-trace-py/pull/3175 should remove this ocncern